lib/modulesync/cli.rb

```diff
@@ -1,10 +1,12 @@
 require 'thor'
+
 require 'modulesync'
+require 'modulesync/cli/thor'
 require 'modulesync/constants'
 require 'modulesync/util'
 
 module ModuleSync
-  class CLI
+  module CLI
```

Comment:
just curious, can you explain why you switched from a class to a module?

Reply:
`CLI` is never instantiated and is only here as a namespace, so switching to module is, for me, a good ruby coding practice.